PROJECT_LIFECYCLE.md

```diff
@@ -155,6 +155,31 @@ You should use the [![Archived badge](https://img.shields.io/badge/Stage-Archive
 [![Archived badge](https://img.shields.io/badge/Stage-Archived-orange)](https://github.com/eiffel-community/community/blob/master/PROJECT_LIFECYCLE.md#stage-archived)
 ```
 
+#### Process for archiving a repository
+
+- Create ticket in the repository to be archived. See [This repo should be archived eiffel-store#10](https://github.com/eiffel-community/eiffel-store/issues/10) for an example
```

Comment:
Do we think "issue" and "ticket" are sufficiently synonymous that we can use the latter term here and only here?